proprietary/tokens/README.md

```diff
@@ -1,3 +1,229 @@
 <!-- @license CC0-1.0 -->
 
-# TODO: write readme
+# Amsterdam Design System: Tokens
+
+This package provides all design tokens from the [Amsterdam Design System](https://designsystem.amsterdam).
+Use it to apply its visual design to your website or application.
+The tokens are pure CSS, making them compatible with most if not all technology stacks.
+
+## About
+
+Every design token is a variable representing a single visual design choice.
+Tokens exist for colours, various aspects of text, spacing lengths, border widths, the configuration of animations, and more.
+The value of every token is valid in CSS.
+
+Some examples:
+
+```css
+:root {
+  --ams-color-blue: #004699;
+  --ams-text-font-weight-bold: 800;
+  --ams-border-width-medium: 0.125rem;
+  --ams-button-padding-inline: 1.5rem;
+  --ams-space-xl: clamp(2.25rem, 2.0625rem + 0.9375vw, 3rem);
+  --ams-text-input-invalid-border-color: var(--ams-color-red);
+}
+```
+
+Tokens start their name with a prefix of `ams-` in CSS.
+The name of a component token (see below) ends with the property that uses it, e.g. `-font-size`.
+
+All tokens together form a theme that encodes the entire branding of the City of Amsterdam.
+
+## Installation
+
+Use only this package if you want or need to reference the tokens directly in your project.
+
+Note that our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) provide more functionality and apply the tokens automatically.
+You should use these packages if your application uses React, or if it allows applying our HTML classes.
+
+```sh
+npm i @amsterdam/design-system-tokens
+```
+
+```sh
+pnpm add @amsterdam/design-system-tokens
+```
+
+```sh
+yarn add @amsterdam/design-system-tokens
+```
+
+## Usage in CSS
+
+Tokens are typically used as custom properties in CSS.
+
+### Main stylesheet
+
+This package offers a main stylesheet containing all tokens.
+They are declared as global CSS variables through the `:root` selector.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+```
+
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.theme.css";
+```
+
+```html
+<html class="ams-theme">
+  …
+</html>
```

Comment:
Setting a class on `html` is a bit unconventional. It is allowed in HTML5 (it wasn't in HTML4), but at best you're styling the `head`, which doesn't have any visual styling 😁 

Setting a global class on `body` makes more sense, imo:

```suggestion
<body class="ams-theme">
  …
</body>
```